hd/lang/lex_utf8.txt

```diff
@@ -1,3 +1,28 @@
+
+    delete/deleted
+en: delete/deleted
+fr: suppression des/supprimés
+
+    delete ancestors/deleted ancestors
+en: delete ancestors/deleted ancestors
+fr: suppression des ascendants/ascendants supprimés
+
+    confirm killing
```

Comment:
I suggest we run the lex tool when we reach distribution. While we update lex messages, it is quite convenient to keep regrouped the new additions.